### PR TITLE
Marcelo/new todo react app

### PR DESCRIPTION
**Pull request by MaxxReis**
* Added new todo-app using react+vite.
* New look for  todo app.
* Renamed old todo-app-react folder to todo-app-react-old

### CODE REVIEW

File: todo-app-react/src/components/modal.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      </Flex>
+      <Flex direction="row">
+        {children}
+      </Flex>
+    </Modal>
+  // {children}
+  )
+}
+
+export default MyModal
```
Comment (MichelDiz) [inline todo-app-react/src/components/modal.jsx:42]: Add new line here

File: todo-app-react/src/Styles/index.js [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+    i{
+        font-size: 24px;
+        color: ${(props) => props.checked ? "#ffffff" : "#100c19"};  
+    }
+`;
+
+export const CopyRight = styled.p`
+    color: ${(props) => props.color || "#85868a"};
+`;
```
Comment (MichelDiz) [inline todo-app-react/src/Styles/index.js:204]: Add new line here

File: todo-app-react/src/GraphQLData.js [hunk 3]
```diff
@@ -100,12 +81,21 @@ export const UPDATE_TODO = gql`
...
+    }) {
+      task {
+        id
+        title
+        completed
+      }
     }
   }
-`;
+`;
```
Comment (MichelDiz) [inline todo-app-react/src/GraphQLData.js:101]: Add new line here

File: todo-app-react/public/vite.svg
Comment (MichelDiz) [inline todo-app-react/public/vite.svg:1]: Remove this or use Dgraph's one.

File: todo-app-react/.gitignore [hunk 4]
```diff
@@ -1,36 +1,30 @@
...
-packages/react-devtools-extensions/firefox/*.xpi
-packages/react-devtools-extensions/firefox/*.pem
-packages/react-devtools-extensions/shared/build
-packages/react-devtools-inline/dist
-packages/react-devtools-shell/dist
+# auth file
+src/auth_template.json
+.env
+package.json
+yarn.lock
```
Comment (MichelDiz) [inline todo-app-react/.gitignore:30]: Add new line here

**Review by MichelDiz — CHANGES_REQUESTED** [review]

File: todo-app-react/src/components/modal.jsx (hunk 1 above)
Comment (MaxxReis) [reply todo-app-react/src/components/modal.jsx]: Solved

File: todo-app-react/src/GraphQLData.js (hunk 3 above)
Comment (MaxxReis) [reply todo-app-react/src/GraphQLData.js]: Solved

File: todo-app-react/public/vite.svg
Comment (MaxxReis) [reply todo-app-react/public/vite.svg:1]: Solved

File: todo-app-react/.gitignore (hunk 4 above)
Comment (MaxxReis) [reply todo-app-react/.gitignore]: Solved

**Pull request closed without merging**